modules/helfi_etusivu_entities/helfi_etusivu_entities.install

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * @file
+ * Contains installation tasks for helfi_etusivu_entities module.
+ */
+
+declare(strict_types=1);
+
+/**
+ * Implements hook_install().
+ */
+function helfi_etusivu_entities_install($is_syncing) : void {
+  // Do not perform following steps if the module is being installed as part
+  // of a configuration import.
+  if ($is_syncing) {
+    return;
+  }
+
+  helfi_etusivu_entities_grant_permissions();
+  helfi_etusivu_entities_create_blocks();
+}
+
+/**
+ * Create required blocks.
+ */
+function helfi_etusivu_entities_create_blocks(): void {
```

Comment:
This doesn't currently run on the Etusivu instance, since this module isn't installed there, which also adds the external entity definitions.

I didn't find any issues with enabling this module locally, but this should probably be tested a bit more and discussed in a daily.

Reply:
Resolved with: https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/642/commits/4640012e214ccf448861e897f2bb2fd52ac2d21b